gensim/matutils.py

```diff
@@ -150,7 +150,7 @@ def zeros_aligned(shape, dtype, order='C', align=128):
     nbytes = np.prod(shape, dtype=np.int64) * np.dtype(dtype).itemsize
     buffer = np.zeros(nbytes + align, dtype=np.uint8)  # problematic on win64 ("maximum allowed dimension exceeded")
     start_index = -buffer.ctypes.data % align
-    return buffer[start_index: start_index + nbytes].view(dtype).reshape(shape, order=order)
+    return buffer[start_index:start_index + nbytes].view(dtype).reshape(shape, order=order)
```

Comment:
Does not conform to PEP8 (need spaces around `:` here, see https://github.com/RaRe-Technologies/gensim/issues/1521#issuecomment-325270209).

Reply:
Sorry, Line slice white space was not my intention. I missed these line in this commit.

Anyway, I agree to input space around `:`.

but if input space like `return buffer[start_index : start_index + nbytes]`, E203 error occurs. [(pycodestyle issue)](https://github.com/PyCQA/pycodestyle/issues/373)

Is it okay upper line?

Reply:
I don't think there's any error in the code, the version with spaces is correct. It appears to be a bug in the style-checking software.